frontend/src/v5/ui/routes/userSignup/userSignUpSSO/userSignUpSSO.component.tsx

```diff
@@ -45,10 +46,16 @@ export const UserSignupSSO = () => {
 	const [contextValue, setContextValue] = useState<UserSignupFormStepperContextValue | null>();
 	const { search } = useLocation();
 	const searchParams = new URLSearchParams(search);
+	const [loginWithSSO,, isLoggingWithSSO] = useSSOLogin();
+
+	if (isLoggingWithSSO) {
+		return null;
+	}
 
 	if (searchParams.get('signupPost')) {
 		if (!searchParams.get('error') || searchParams.get('error') === SSOErrorCode.EMAIL_EXISTS_WITH_SSO) {
```

Comment:
The EMAIL_EXISTS_WITH_SSO error is never actually displayed. Should there be some sign that you've been logged in to the existing account rather than created a new one?

Reply:
It automatically logs in if the emails exists with sso, so no error is meant to be displayed.